bin/gopro2gpx/gopro2gpx.go

```diff
@@ -57,14 +58,17 @@ func main() {
 		telems := t_prev.ShitJson()
 
 		for i, _ := range telems {
+			secs := int64(math.Floor(telems[i].TS))
```

Comment:
These changes don't actually change the output of gopro2gpx _yet_, because [tkrajina/gpxgo](github.com/tkrajina/gpxgo) doesn't support milliseconds in GPX timestamps, but I'm going to create a pull request on that repository to add the ability to choose your timestamp format.